src/Webapi/Webapi__Fetch.resi

```diff
@@ -1,6 +1,6 @@
 module AbortController = Webapi__Fetch__AbortController
 
-/** Alias for anyone upgrading */
+/* * Alias for anyone upgrading */
```

Comment:
The lack of ReScript support for docs comments is annoying. Maybe we should just delete this comment now 🤷 and arguably the alias, too, since this will be going onto the 2.0 branch

Reply:
I'm happy to delete the aliases in a followup PR when all of these are merged as I don't fancy like dealing with the conflicts 😆